.eslintrc.json

```diff
@@ -41,6 +41,11 @@
     }
   ],
   "rules": {
+    "no-param-reassign": ["warn"],
```

Comment:
Best practice is discuss any rule turn-off with team. If you did that than this comment is irrelevant.